pkg/ccrm/ccrm.go

```diff
@@ -59,6 +59,8 @@ type Flags struct {
 	DnsZone                       string
 	CloudletRegistryPath          string
 	CloudletVMImagePath           string
+	EnvoyWithCurlImage            string
```

Comment:
I would suggest just renaming them to envoyImage and NginxImage to be more generic

Reply:
Hm, but wouldn't that invite confusion? If someone specifies the latest upstream envoy, things are going to fail because it doesn't have curl built into it. At least with this name it's clear that this is not just a vanilla envoy image.

Reply:
From the code, it looks like we use curl for unix domain socket case only, which is an e2e/unit test case. We can keep the "withCurl" note, since we are using it for now, and if we change it at some point not to need curl at all, we can change the var names.